### DISCUSSION TITLE
Allow modifying ignored files for build

### DISCUSSION THREAD

**Discussion by darcien** [callstack/react-native-builder-bob#447] [Ideas]
Right now bob has hardcoded ignore pattern when building:
https://github.com/callstack/react-native-builder-bob/blob/6650bc170b3c872ad290965856162334c8394df6/packages/react-native-builder-bob/src/utils/compile.ts#L34

Not all projects follow this pattern.
I would like to be able to modify this ignore pattern without patching the sources.

This has been raised multiple times with no clear responses or solutions:
- https://github.com/callstack/react-native-builder-bob/issues/197
- https://github.com/callstack/react-native-builder-bob/issues/162
- https://github.com/callstack/react-native-builder-bob/discussions/325

**Comment by atlj** [comment]
We can leverage the bob config to make it possible to customize it. Will see what can be done.

**Reply by atlj** [reply]
Please see https://github.com/callstack/react-native-builder-bob/pull/450

**Reply by darcien** [reply]
That PR looks perfect, thank you!

**Comment by atlj** [comment]
Closing this as #450 is merged.
